howtos/how-to-make-clickable-tabs-for-conversations/styles.css

```
@@ -32,24 +32,26 @@
     cursor: pointer;
     padding: 14px 16px;
     transition: 0.3s;
-    font-size: 17px;
+    font-size: 14px;
   }
   
   /* Change background color of buttons on hover */
   .tab button:hover {
-    background-color: #ddd;
+    background-color: #f7f7f7;
```

Suggestion to visually distinguish the hover

```suggestion
    background-color: #ececec;
```